src/figuresClasses.ts

```diff
@@ -2,10 +1,91 @@
+export interface Figure {
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  shape: string = 'triangle';
+
+  color: string;
 
-export class Triangle implements Figure {}
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Length of some value is less than 0');
+    }
+
+    const arrSides = [a, b, c].sort((one: number, two: number) => two - one);
+    const maxSide = arrSides[0];
+    const sumSides = arrSides[1] + arrSides[2];
+
+    if (maxSide >= sumSides) {
+      throw new Error(`${maxSide} of triangle >= than a sum of two others`);
+    }
+
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const p = (this.a + this.b + this.c) / 2;
+
+    return +Math.sqrt(p * (p - this.a) * (p - this.b) * (p - this.c)).toFixed(
+      2,
+    );
+  }
+}
+export class Circle implements Figure {
+  shape: string = 'circle';
 
-export class Circle implements Figure {}
+  color: string;
 
-export class Rectangle implements Figure {}
+  radius: number;
+
+  constructor(color: string, radius: number) {
+    if (radius <= 0) {
+      throw new Error('Length of some value is less than 0');
```

Comment:
The error message 'Length of some value is less than 0' is misleading. It should be 'Length of some value is less than or equal to 0' to accurately reflect the condition being checked.